integration/src/main/resources/configs/integration/DatasetOperations.json

```diff
@@ -4,6 +4,7 @@
   "serverSpecificationFile": "catalog-local.json",
   "kubernetes": {},
   "application": {},
+  "maxRetries": 0,
```

Comment:
The newer terra test-runner supports retries and has a built-in default of 3 retries. Since our returnable failures don't usually occur inside the test, but inside setup or shutdown, extra retries are unlikely to help and will instead make real failures take longer to detect.